client/base/help_options.cc

```diff
@@ -88,14 +88,14 @@ void Mysql::Tools::Base::Options::Help_options::print_usage()
 
   if (first_year_str == COPYRIGHT_NOTICE_CURRENT_YEAR)
   {
-    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE(COPYRIGHT_NOTICE_CURRENT_YEAR);
```

Comment:
Why 2021?
MySQL for 5.7: https://github.com/mysql/mysql-server/blob/5.7/client/base/help_options.cc#L98

Reply:
because atomstone was founded in 2021，so,plan to start from 2021

Reply:
make sense.